src/versionSocketConnection.ts

```diff
@@ -79,7 +80,7 @@ export class BrowserVersionDetectionSocket extends EventEmitter {
                 return {revision: currentRevision, isHeadless};
             }
             if (currentVersion[i] < minSupportedVersion[i]) {
-                return {revision: '', isHeadless};
+                return {revision: MIN_SUPPORTED_REVISION, isHeadless};
```

Comment:
We may want to another check for Edge 101 and display an upgrade prompt/error message if the target is below it, since some extension functionality was moved to dtfe.

Reply:
Thanks. Anything you can think of in CDP that would be a problem here? I tested this forced upgrade with both Edge 101 and Edge 100 - DevTools worked fine for both since we're now using the 102 dt-fe code in either case. Barring a breaking CDP change I think we should be able to leave this as-is.

Reply:
Nope. CDP compat should be fine for a few major browser versions back.